membership-baseline.md

```diff
@@ -0,0 +1,169 @@
+# Overview:
+The null Community operates on a decentralized model where participants play essential roles as volunteers, core team members, and board members. While the community does have a membership program, it's important to note that the majority of our resources and activities are accessible to both members and non-members alike.
+
+## Membership
+The null Community offers a membership program that provides additional benefits and opportunities for active participants. Members have the privilege of serving as core team members, volunteers, and potentially as board members. However, it's worth emphasizing that membership is not a prerequisite for engaging in most of the community's activities.
+
+## Engagement and Access
+We strive to keep our community inclusive and open to all individuals and organizations. Non-members are encouraged to participate in projects, attend meetings, join mailing lists, and take part in events. Most of our resources and activities are readily available to everyone, fostering a collaborative and accessible environment.
+
+## Membership Benefits
+While most resources are accessible to non-members, becoming a member of the null Community provides additional benefits such as enhanced involvement in core team activities, increased networking opportunities, and the ability to shape the community's direction through potential board membership. Membership also signifies a deeper commitment and dedication to the community's goals and values.
+
+# Membership Benefits:
+At null, we offer a range of exclusive benefits to our members, including:
+
+**Access to Cutting-Edge Security Resources:**
+
+- Online library of security research papers, articles and access to hundreds of events and their resources
+    - https://github.com/null-open-security-community/Documentation/tree/master/null-Research-Papers
+    - https://blog.null.community/
+    - https://null.community/archives
+- Professional mentoring programs
+- Priority access to null Study groups
+    - Cloud Security
+    - Crypto
+    - Web application security
+    - Container security(WIP)
+- Few other work in progress benefits:
+    - Access to the GitHub Student Pack
+    - Visual Studio Benefits, providing you with valuable resources for your security projects.
+    - One month free pluralsight membership
+    - Free access to security training programs by - <Partner with learning platforms/companies and ask for free training subscriptions>
+- <Add more benefits here> related to learning resources
+
+**Note:** We can not promise the work in progress benefits as it mainly depends on the vendor and their community partnership policy.
```

Comment:
exactly so should we keep them here.

Reply:
Lets keep it here and we can put a WIP or something to say that we yet provide it or not

Modified 

- Few other benefits we plan to included (WIP):
    - Access to the GitHub Student Pack
    - Visual Studio Benefits, providing you with valuable resources for your security projects.
    - One month free pluralsight membership
    - Free access to security training programs by - <Partner with learning platforms/companies and ask for free training subscriptions>
- <Add more benefits here> related to learning resources

**Note:** We can not promise the work in progress benefits as it mainly depends on the vendor and their community partnership policy. This may or may not happen.